.github/workflows/python-dockers.yml

```diff
@@ -0,0 +1,38 @@
+# Workflow to build example Docker images with TileDB-SOMA installed
+name: TileDB-SOMA docker builds
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: "42 9 * * *"
+  pull_request:
+    paths:
+      - .github/workflows/python-dockers.yml
+      - docker/**
+jobs:
+  docker:
+    strategy:
+      matrix:
+        os:
+          - ubuntu-24.04
+          - ubuntu-24.04-arm
+          # TODO: currently impossible(?) to build Docker images on macOS runners
```

Comment:
Please file a tracking task (GitHub issue) and include its URL here

Reply:
Done: #3781